tests/core/parameter_validation/test_parameter_validation.py

```diff
@@ -49,11 +49,30 @@ def test_filesystem_hierarchy():
     path = os.path.join(BASE_DIR, 'filesystem_hierarchy')
     parameters = ParameterNode('', directory_path = path)
     parameters_at_instant = parameters('2016-01-01')
-    assert parameters_at_instant.node1.param == 1.0
+    assert_equals(parameters_at_instant.taxes.rate, 0.22)
 
 
 def test_yaml_hierarchy():
     path = os.path.join(BASE_DIR, 'yaml_hierarchy')
     parameters = ParameterNode('', directory_path = path)
     parameters_at_instant = parameters('2016-01-01')
-    assert parameters_at_instant.node1.param == 1.0
+    assert_equals(parameters_at_instant.taxes.rate, 0.22)
+
+
+def test_parameters_metadata():
+    path = os.path.join(BASE_DIR, 'filesystem_hierarchy')
```

Comment:
Why not use the country template for these tests ?
e.g.
```py
import openfisca_country_template
tbs = openfisca_country_template.CountryTaxBenefitSystem()
basic_income = tbs.parameters.benefits.basic_income
assert_equals(basic_income.description, 'Amount of the basic income')
```

Reply:
Done